src/fetchtastic/download/firmware.py

```
@@ -2035,6 +2076,21 @@ def download_repo_prerelease_firmware(
                 latest_release_tag, active_dir, cache_manager=self.cache_manager
             )
+        if dirs_to_track and self.config.get(
+            "CREATE_LATEST_SYMLINKS", DEFAULT_CREATE_LATEST_SYMLINKS
+        ):
+            try:
+                update_latest_pointer(
+                    prerelease_base_dir,
+                    dirs_to_track[-1],
+                    LATEST_POINTER_NAME,
+                )
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)